src/events/guildMemberUpdate.js

```diff
@@ -20,6 +21,11 @@ async function guildMemberUpdateHandler(oldMember, newMember) {
 
 	const latestLog = auditLogs.entries.first();
 
+	if (latestLog == guild.fetchAuditLogs({
+		limit: 1,
+		type: 'MEMBER_ROLE_UPDATE'
+	})) return;
+
```

Comment:
What is the point of this here? It's not clear what this fixes. Also this `if` statement is formatted in a very non-readable way

Reply:
I have updated this into one line, making it a skip instead of an entire check function.